docs/RELEASE.md

```diff
@@ -67,6 +88,8 @@ git commit -a -m "Update project version"
 
 ## Tag SDK
 
+> As noted previously, wait with the tagging until the prerequisite Debian packages has been published.
```

Comment:
Could you please add a little more detail what is needed here, and ideally a link to the previous notes?

Reply:
Yes, added more elaboration and backlinks to the same document.